uber/templates/static_views/prereg_not_yet_open.html

```diff
@@ -13,7 +13,7 @@ <h1 style='color:red'>{{ c.EVENT_NAME_AND_YEAR }} pre-registration is not yet op
   {%- if c.DEALER_REG_START and c.DEALER_REG_PUBLIC and c.BEFORE_DEALER_REG_SHUTDOWN -%}
   <p>
     <strong>Dealers:</strong>
-    {% if c.BEFORE_DEALER_REG_PUBLIC %}
+    {% if c.BEFORE_DEALER_REG_START %}
```

Comment:
The intended use of these variables is not clear. `dealer_reg_public` defaults to the same value as `dealer_reg_start`. From what I can gather, `dealer_reg_public` will always either be the same as `dealer_reg_start` or an empty value to entirely turn off dealer registration visibility. If you notice, the outer conditional is checking the truthiness of `c.DEALER_REG_PUBLIC` to completely hide this block of HTML.

Let's assume for the moment that both `c.DEALER_REG_START` and `c.DEALER_REG_PUBLIC` are truthy values (as we expect they will be for most configurations, unless we are completely hiding dealer reg – in which case `c.DEALER_REG_PUBLIC` would be falsey, but we've already discussed how that is handled by the outer conditional). 

So both `c.DEALER_REG_START` and `c.DEALER_REG_PUBLIC` are truthy. There are three states we can have in this scenario:
* `c.DEALER_REG_START == c.DEALER_REG_PUBLIC` in which case, this distinction is moot
* `c.DEALER_REG_START < c.DEALER_REG_PUBLIC` dealer reg starts before it's public
* `c.DEALER_REG_START > c.DEALER_REG_PUBLIC` dealer reg is public before it starts

# `c.DEALER_REG_START < c.DEALER_REG_PUBLIC`

In this scenario, if a user loads the page in the time AFTER `c.DEALER_REG_START` but BEFORE `c.DEALER_REG_PUBLIC`, then they'll receive a message saying that dealer registration began some time in the past, but they'll have no way to navigate to the dealer application page.

This is obviously not what we want.

# `c.DEALER_REG_START > c.DEALER_REG_PUBLIC`

In this scenario, if a user loads the page in the time AFTER `c.DEALER_REG_PUBLIC` but BEFORE `c.DEALER_REG_START`, then they'll receive a link that says "Please register here". Upon clicking that link, they'll be redirected back to the same exact page because we haven't started accepting dealer registrations yet.

This is also obviously not what we want.

Thus, the correct value to check for that conditional is `c.BEFORE_DEALER_REG_START`.